packages/beacon-node/src/execution/builder/http.ts

```diff
@@ -128,8 +132,17 @@ export class ExecutionBuilderHttp implements IExecutionBuilder {
     }
   }
 
-  async registerValidator(registrations: bellatrix.SignedValidatorRegistrationV1[]): Promise<void> {
+  async registerValidator(epoch: Epoch, registrations: bellatrix.SignedValidatorRegistrationV1[]): Promise<void> {
```

Comment:
is adding epoch to api standard spec now?

Reply:
This is an internal method, the api call happens on [L136](https://github.com/ChainSafe/lodestar/blob/3b4f655b0867e2a507dc8ca0410088f3e8e4e675/packages/beacon-node/src/execution/builder/http.ts#L136) which does not need the epoch, we only need it to track registrations in our internal cache to be able to prune old records (similar to what we do for proposer cache).

I opted for supplying the epoch by the caller of `registerValidator` but it could also just use `clock.currentEpoch`